image/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM openjdk:17-jdk-slim
+
+RUN apt update && \
+    apt install -y unzip wget && \
+    mkdir /grasscutter && cd /grasscutter && \
+    wget -O grasscutter.jar https://github.com/Grasscutters/Grasscutter/releases/download/v1.4.3/grasscutter-1.4.3.jar && \
+    wget https://raw.githubusercontent.com/Grasscutters/Grasscutter/development/keystore.p12 && \
+    wget -P /grasscutter/resources https://git.crepe.moe/grasscutters/Grasscutter_Resources/-/archive/main/Grasscutter_Resources-main.zip && \
+    unzip -d /grasscutter/resources /grasscutter/resources/Grasscutter_Resources-main.zip && rm /grasscutter/resources/Grasscutter_Resources-main.zip && \
+    unzip -d /grasscutter/resources /grasscutter/resources/Grasscutter_Resources-main/Grasscutter_Resources-3.3.zip && rm -rf /grasscutter/resources/Grasscutter_Resources-main && \
+    mv /grasscutter/resources/Grasscutter_Resources-3.3/Resources/* /grasscutter/resources/ && rm -rf /grasscutter/resources/Grasscutter_Resources-3.3 && \
+    apt remove -y unzip wget
+
+EXPOSE 443
```

Comment:
These should not be hardcoded values, as they can be configured in the configuration file. (if need be, add something to keep the config consistent, again not familiar with how `Dockerfile` works)

Reply:
@closetool 
The `EXPOSE` instruction does not actually publish the port. It functions as a type of documentation between the person who builds the image and the person who runs the container, about which ports are intended to be published.
[Dockerfile reference](https://docs.docker.com/engine/reference/builder/#expose)

I suggest these can be removed as the ports are documented in the wiki and the configuration file